include/opentracing/tracer.h

```diff
@@ -84,7 +84,7 @@ class OPENTRACING_API Tracer {
   //     auto span = tracer.StartSpan(
   //         "GetFeed",
   //         {opentracing::ChildOf(&parentSpan.context()),
-  //         opentracing::Tag{"user_agent", loggedReq.UserAgent},
+  //         opentracing::SetTag{"user_agent", loggedReq.UserAgent},
```

Comment:
I think the intention of this comment was actually correct, it's the class that's misnamed. Changing class name is a breaking change, but could we deprecated it and create a new one called `Tag`?

Reply:
Or we could simplify everyone's life massively by living with the slightly misnamed class ;)